src/configs/ubiquibot-config-default.ts

```diff
@@ -0,0 +1,102 @@
+import { MergedConfig } from "../types";
+
+export const DefaultConfig : MergedConfig = {
+  evmNetworkId: 100,
+  priceMultiplier: 1,
+  issueCreatorMultiplier: 2,
+  paymentPermitMaxPrice: 9007199254740991,
+  maxConcurrentAssigns: 9007199254740991,
+  assistivePricing: false,
+  disableAnalytics: false,
+  commentIncentives: false,
+  registerWalletWithVerification: false,
+  promotionComment: "\n<h6>If you enjoy the DevPool experience, please follow <a href='https://github.com/ubiquity'>Ubiquity on GitHub</a> and star <a href='https://github.com/ubiquity/devpool-directory'>this repo</a> to show your support. It helps a lot!</h6>",
+  defaultLabels: [],
+  timeLabels: [
+
+    {
+      name: "Time: <1 Hour",
+    },
+    {
+      name: "Time: <1 Day",
+    },
+    {
+      name: "Time: <1 Week",
+    },
+    {
+      name: "Time: <2 Weeks",
+    },
+    {
+      name: "Time: <1 Month",
+    },
+  ],
+  priorityLabels: [
+    {
+      name: "Priority: 1 (Normal)",
+    },
+    {
+      name: "Priority: 2 (Medium)",
+    },
+    {
+      name: "Priority: 3 (High)",
+    },
+    {
+      name: "Priority: 4 (Urgent)",
+    },
+    {
+      name: "Priority: 5 (Emergency)",
+    },
+  ],
```

Comment:
I see that there are no `value` or `weight` properties which currently exists in our organization wide config. Do you mind taking a look at the organization wide config, and removing the extraneous properties?

Reply:
I don't see `value` or `weight` properties in [org config](https://github.com/ubiquity/ubiquibot-config/blob/development/.github/ubiquibot-config.yml)